src/client/datascience/jupyter/pythonVariableRequester.ts

```diff
@@ -29,13 +29,9 @@ export class PythonVariablesRequester implements IKernelVariableRequester {
         await this.importDataFrameScripts(notebook);
 
         // Then execute a call to get the info and turn it into JSON
-        const results = await notebook.execute(
-            `print(${DataFrameLoading.DataFrameInfoFunc}(${expression}))`,
-            Identifiers.EmptyFileName,
-            0,
-            uuid(),
-            undefined,
-            true
```

Comment:
This is the silent parameter that is still on the notebook.execute. However with a recent change it doesn't do anything anymore. I'll actually go and remove it from the interface quick so we don't do this again on accident.

Reply:
Updated.